### PR TITLE
Feature/kahftube

### PR DESCRIPTION
**Pull request by asifddlks**
Task/Issue URL: [https://github.com/AsilHQ/Android/issues/9](url)

### Description
Implemented kahftube feature with static authentication (not google account).

@denizgencay

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt
Comment (denizgencay) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:1]: Could you please use Timber for every log?

File: app/src/main/java/com/duckduckgo/app/kahftube/SharedPreferenceManager.kt
Comment (denizgencay) [inline app/src/main/java/com/duckduckgo/app/kahftube/SharedPreferenceManager.kt:1]: Could you please change value force to null check?

File: app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt
Comment (asifddlks) [reply app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt:1]: That callHandler method log will be removed after completing the login prompt part.
And yes we should use timber. Moreover we should have our extension (please check code snippet) to have more control on common things.

`fun Context.showLog(msg: String) {
    Timber.tag(this.javaClass.simpleName).d(msg)
}

fun Context.showErrorLog(msg: String) {
    Timber.tag(this.javaClass.simpleName).e(msg)
}`

File: app/src/main/java/com/duckduckgo/app/kahftube/SharedPreferenceManager.kt
Comment (asifddlks) [reply app/src/main/java/com/duckduckgo/app/kahftube/SharedPreferenceManager.kt:1]: I had used DataStore Preference as it's now recommended by google. But here I have seen you have used SharedPreference in SafeGaze implementation. So I made a SharedPreferenceManager real quick. 
I have updated the codes. Please check.

**Review by denizgencay — APPROVED** [review]
Looks good to me

**Pull request merged**